gobblin-metrics-libs/gobblin-metrics/src/main/java/org/apache/gobblin/metrics/ServiceMetricNames.java

```diff
@@ -37,5 +37,5 @@
   public static final String RUN_IMMEDIATELY_FLOW_METER = "RunImmediatelyFlow";
 
   public static final String RUNNING_FLOWS_COUNTER = "RunningFlows";
-  public static final String FLOWS_ALREADY_RUNNING_GAUGE = "FlowsAlreadyRunning";
+  public static final String COMPILED = "Compiled";
```

Comment:
I think 'Compiled' is ok, because we are appending flow details before it.
e.g. ktwo.encryption_holdem_faro.Compiled = 1/0 seems better than
ktwo.encryption_holdem_faro.CompiledFlows = 1/0